cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/CaReconciler.java

```diff
@@ -372,8 +373,38 @@ Future<Void> rollingUpdateForNewCaKey() {
         }
     }
 
-    // ZooKeeper is rolled only for new Cluster CA key
-    private Future<Void> maybeRollZookeeper(RestartReasons podRestartReasons) {
+    /**
+     * If we need to roll the ZooKeeper cluster to roll out the trust to a new CA private key (i.e. when a CA private
```

Comment:
"to roll out the trust to a new CA private key (i.e. when a CA private key is replaced)" ... I guess it should be "to roll out the trust to a new CA certificate (i.e. when a CA private key is replaced) because you are "trusting" the new certificate (coming because of a new key) but not the new key itself.

Reply:
Better?